### PR TITLE
[DEV-817] Add prettier and eslint

### PR DESCRIPTION
**Pull request by katrinundfritz**

### CODE REVIEW

File: src/components/preconfigured/Standard.jsx [hunk 1]
```diff
@@ -3,40 +3,54 @@ import styled from 'styled-components'
...
-    width="230"
-    frameBorder="0"
-    src={`https://streambot.betterplace.org/fundraising-events/${eventId}/${widget}?textAlign=left&textColor=fff&fontFamily=Fira+Sans&headline=false${demo ? '&demo=true' : ''}`}
-  />
-</FluidColumn>
+const DataBlock = ({ children, widget, eventId, demo, headlineStyle }) => (
+  <FluidColumn>
+    <Headline style={headlineStyle}>{children}</Headline>
+    <iframe
+      title={widget}
```
Comment (josefie) [inline src/components/preconfigured/Standard.jsx:37]: You probably checked that, but just to make sure: We don't depend on the name attribute anywhere, right?

Comment (katrinundfritz) [reply src/components/preconfigured/Standard.jsx:37]: I checked and couldn't find something that depends on it.

**Review by josefie — APPROVED** [review]

**Review by smnscp — APPROVED** [review]

**Comment by smnscp** [comment]
@katrinundfritz, can we squash-merge this and rebase #67?

**Comment by katrinundfritz** [comment]
It is in a state where it cannot be deployed. So I would only do it when #67 is ready to be merged, too. I don't want to block the master.

**Pull request merged**